### PR TITLE
add option to configure resource settings

### PR DESCRIPTION
**Pull request by etam**
It depends on #12 (the target branch is not master, to show only relevant changes)

It's not a perfect solution (it requires a running golem). I made a proposal to make it better: https://github.com/golemfactory/golem/issues/3570

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -197,3 +197,19 @@
     name: "{{ golem_source_unit_name }}"
     state: started
     enabled: yes
+
+- name: configure resources
+  when: golem_source_resource_settings
+  block:
+    - name: waiting for golem to become ready
+      shell: "{{ golem_source_cli }} account info | grep Golem_ID | awk '{print $2}'"
```
Comment (ktomala) [inline tasks/main.yml:205]: There's no golem_source_cli variable in defaults.yml

File: tasks/main.yml [hunk 2]
```diff
@@ -197,3 +197,19 @@
...
+      changed_when: False
+      register: golem_source_node_id
+      until: golem_source_node_id.stdout != ""
+      # generating keys and downloading docker images might take a while
+      retries: 10
+      delay: 60
+
+    - name: configure resources
+      shell: "{{ golem_source_cli }} settings set {{ item.key }} {{ item.value }}"
```
Comment (ktomala) [inline tasks/main.yml:214]: No golem_source_cli variable in defaults

**Review by ktomala — CHANGES_REQUESTED** [review]
golem_source_cli variable is missing

File: tasks/main.yml (hunk 1 above)
Comment (etam) [reply tasks/main.yml:205]: it's in vars

Comment (etam) [reply tasks/main.yml:205]: it's added by #12

**Review by ktomala — APPROVED** [review]

**Pull request merged**